src/main/kotlin/com/openlattice/assembler/AssemblerConnectionManager.kt

```diff
@@ -809,9 +793,11 @@ internal fun createRoleIfNotExistsSql(dbRole: String): String {
             "      FROM   pg_catalog.pg_roles\n" +
             "      WHERE  rolname = '$dbRole') THEN\n" +
             "\n" +
-            "      CREATE ROLE ${ApiHelpers.dbQuote(
-                    dbRole
-            )} NOSUPERUSER NOCREATEDB NOCREATEROLE NOINHERIT NOLOGIN;\n" +
+            "      CREATE ROLE ${
+                ApiHelpers.dbQuote(
+                        dbRole
+                )
```

Comment:
maybe it's just me but I feel like this should end up as `CREATE ROLE ${ApiHelpers.dbQuot( dbRole )` all on one line

Reply:
yeah that's prettier -- it requires manual updates but it looks like the autoformatter doesn't undo it so I'm down